scripts/php/generate-api-client.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source ../setup-variables.sh
+
+CLI_JAR="${SFMC_SWAGGER_ROOT_FOLDER}/cli/swagger-codegen-cli-2.4.7.jar"
```

Comment:
Since the newer version may contain bug fixes I suggest we upgrade. On the other hand I don't see any reason why we should use the same version since the user does not interact with the generated code and all the generated code passes the tests.